### DISCUSSION TITLE
Google web crawling cannot find my robots.txt file

### DISCUSSION THREAD

**Discussion by Nacabacu** [vercel/community#7296] [Help]
I have a website built with Next.js and deployed to Vercel. I added the robots.txt files followed [this document](https://nextjs.org/docs/app/api-reference/file-conventions/metadata/robots).

However, after I added my site to Google Search Console, it shows that cannot find robots.txt on my site. It makes the search result from Google is different from other search engine (bing, qwant) as the other search engine seemed to be able to find the robots.txt and sitemap.xml and show search result on ever page added in the sitemap.xml as expected.

website: punn.dev
robots: punn.dev/robots.txt
sitemaps: punn.dev/sitemap.xml

I'm not sure if anyone has found the same issue. Thank you in advance for the help.

**Comment by versecafe** [comment]
@Nacabacu make sure you point it at specifically `/robots.txt` as `/robots` will return a `404` also you can always add a sitemap route directly at `https://search.google.com/search-console/sitemaps` and do note depending on the version of NextJS you're using `/sitemap.xml` may error out as in the newest canaries you now need to target `/sitemap`

**Reply by Nacabacu** [reply]
Yeah, I specifically pointed at the /robots.txt file. I also manually added the /sitemap.xml file on the google search console, it found the sitemap but the there's no page index and search result is still the same after added it a few days.

![image](https://github.com/vercel/community/assets/31649500/d68a8a43-e0fb-4c85-966f-4af51a390164)

[a comment by an automated account is not shown]
